srv/wordpress/wp-content/plugins/acore-wp-plugin/src/Manager/Soap/GuildService.php

```diff
@@ -0,0 +1,14 @@
+<?php
+
+namespace ACore\Manager\Soap;
+
+use ACore\Manager\Soap\AcoreSoapTrait;
+
+class GuildService {
+
+    use AcoreSoapTrait;
+
+    public function guildRename($oldname, $newname) {
+        return $this->executeCommand(".guild rename \"$oldname\" \"$newname\"");
```

Comment:
If this is the standard format of passing commands to the game server, I recommend filtering out `"` characters from inputs on php side.